src/test/java/org/openrewrite/java/testing/mockito/CleanupMockitoImportsTest.java

```diff
@@ -97,6 +97,33 @@ void test() {
         );
     }
 
+    @Test
+    void doNotRemoveImportsPossiblyAssociatedWithAnUntypedMockitoMethodMixed() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import org.mockito.Mockito;
+              import static org.mockito.Mockito.when;
+              import static org.mockito.BDDMockito.given;
+              import static org.mockito.Mockito.verifyNoInteractions;
+
+              class MyObjectTest {
+                MyObject myObject;
+                MyMockClass myMock;
+                            
+                void test() {
+                  when(myObject.getSomeField()).thenReturn("testValue");
+                  given(myObject.getSomeField()).willReturn("testValue");
```

Comment:
These get methods are not defined, leading to a missing types and hence the incorrect conclusion that there are unused imports. Does the test still fail if these get methods are present?

Reply:
I think this is a deeper problem in the RemoveImports class. in the evaluation of maybeRemove the non-static import, it sees the static when import, and removes that instead

Reply:
I tried repro in the unit test for RemoveImport and it didn't repro, so there is another factor im missing. Back to debugging...

Reply:
Much appreciated that you're having a detailed look!

Reply:
ok my unit test in RemoveImport CAN repro the issue when i put an "unknown" parameter into the static method call. so your hunch about that was right. But i think maybe the fix is that the mockito recipe should not even be attempting to remove the class import in this case

Reply:
OK I think I have a solution. what do you think?

Reply:
Looks good! Thanks for working through this one. I've added a very minor change to use TypeUtils, such that it's more inline with what we use elsewhere, but otherwise happy to see this resolved!